microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots.md

```diff
@@ -45,136 +46,140 @@ This article uses five demo webpages, all sourced at https://github.com/Microsof
 
-| Constructor (group) entry | Description |
+To reveal the list of objects that were instantiated by a given constructor, expand the constructor group.
+
+###### Special category names in the Summary view
+
+The **Summary** view also contains special category names that aren't based on constructors. These special categories are:
+
+<!-- Taken from https://github.com/sethbrenith/sethbrenith.github.io/blob/main/heap-snapshot-names.md -->
+| Category name | Description |
 |:--- |:--- |
-| **(global property)** | The intermediate objects between a global object (like `window`) and an object referenced by it.  If an object is created using a constructor `Person` and is held by a global object, the retaining path would be represented as `[global] > (global property) > Person`.  This contrasts with the norm, where objects directly reference each other.  Intermediate objects exist for performance.  Globals are modified regularly and property access optimizations do a good job for non-global objects aren't applicable for globals.  |
-| **(roots)** | The root entries in the retaining tree view are the entities that have references to the selected object.  These can also be references created by the engine for its own purposes.  The engine has caches which reference objects, but all such references are weak and don't prevent an object from being collected, given that there are no truly strong references.  |
-| **(closure)** | A count of references to a group of objects through function closures. |
-| **(array, string, number, regexp)** | A list of object types with properties which reference an Array, String, Number, or regular expression. |
-| **(compiled code)** | Everything related to compiled code.  Script is similar to a function, but corresponds to a `<script>` body.  SharedFunctionInfos (SFI) are objects standing between functions and compiled code.  Functions usually have a context, while SFIs do not. |
-| **HTMLDivElement**, **HTMLAnchorElement**, **DocumentFragment**, and so on.  | References to elements or document objects of a particular type referenced by your code. |
-| **(object shape)** | References to the hidden classes and descriptor arrays that V8 (the JavaScript engine of Microsoft Edge) uses to understand and index the properties in objects.  See [HiddenClasses and DescriptorArrays](https://v8.dev/blog/fast-properties#hiddenclasses-and-descriptorarrays). |
-| **(BigInt)** | References to the **BigInt** object, which is used to represent and manipulate values that are too large to be represented by the **Number** object.  See [BigInt](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/BigInt). |
-
-
-<!--
-### Heap profiling summary
-todo: add heap profiling summary section when available
--->
+| **(array)** | Various internal array-like objects which don't directly correspond to objects visible from JavaScript, such as the contents of JavaScript Arrays, or the named properties of JavaScript objects. |
```

Comment:
```suggestion
| **(array)** | Various internal array-like objects that don't directly correspond to objects visible from JavaScript, such as the contents of JavaScript Arrays, or the named properties of JavaScript objects. |
```
i'd probably use "which", but allegedly it should be "that", unless add comma.
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which